### PR TITLE
Recommended way to learn commands

### LINKED ISSUE #110: Recommend tldr
I find that [tldr](https://tldr.sh/) is much better than the long man page.

I think we should recommend tldr somewhere.

### PR DESCRIPTION
**Pull request by seisman**
Closes #110.

### CODE REVIEW

File: source/computer/commands.rst [hunk 1]
```diff
@@ -1,8 +1,33 @@
...
+        - Create directories recursively (useful for creating nested dirs):
+            mkdir -p path/to/directory
+
+2.  使用 UNIX 提供的 ``man`` 命令查询
+
+    ``man`` 提供了命令的完整语法，因而通常比较冗长。例如::
+
+        $ man mkdir
+
+3.  网页搜索相关命令
```
Comment (core-man) [inline source/computer/commands.rst:30]: Shall we recommand some websites? e.g.
- google
- https://man.linuxde.net/
- https://tldr.ostera.io/

Comment (seisman) [reply source/computer/commands.rst:30]: Yes, but i haven't found satisfactory sites.

Comment (core-man) [reply source/computer/commands.rst:30]: https://man.linuxde.net
this one is good

Comment (seisman) [reply source/computer/commands.rst:30]: Not as good as I expect, but OK to list it.

Comment (seisman) [reply source/computer/commands.rst:30]: Done in df6923aa2fb9b2e8a0e3369392d5f244708e4846.

**Review by core-man — APPROVED** [review]

**Pull request merged**